### DISCUSSION TITLE
Longitudinal studies and replicates

### DISCUSSION THREAD

**Discussion by nzangger** [IBEXImagingCommunity/ibex_imaging_knowledge_base#62] [General]
Hi Andrea, 

First and foremost, thank you for the amazing work you're putting together to help the imaging community. I would have the following question(s) regarding the **design of longitudinal studies using IBEX**:

1) What would be your thoughts on the number of -plex (how many markers) you would aim for when designing a longitudinal study?
         Here I was thinking of using a higher plex for discovery with fewer time points and a lower plex with more time points when 
         focussing on a phenomenon of interest. 

2) How much per time point would you image per time point (i.e. how many replicates or sections per time point)? 

I'm very much looking forward to your answers.

Thanks,
Nathan

**Comment by radtkea** [comment]
Hi Nathan,

Thanks VERY much for your excellent question and very kind words. We are delighted the Imaging Community is helpful for your studies.

I wish I could give you a definitive answer. Instead, I will leave you with some guiding principles.

1. Imaging, especially highly multiplexed imaging, is low throughput compared to other single cell technologies. 
2. Flow cytometry is our friend! I appreciate that some questions can only be answered by imaging, but flow cytometry (or other technologies) can help you identify key time points for your imaging studies. 
3. I very much like your suggestion to do a higher plex for discovery and a lower plex with more time points/samples for validation. I did this for a recent study. I would expand on this point to suggest doing several pilot low-plex panels on different time points to narrow in on the critical markers and questions for your IBEX experiment. Is is tedious? Yes. But the more pre-work (flow, traditional imaging, other assays) one does, the better the study:)
4. The number of replicates, sections, area/volume to image, etc....all great questions! This depends on so many variables. We tried to summarize these considerations in Figure 2 of this [Perspective](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9264278/). We didn't provide any fast and hard rules:) There aren't any!
5. Historically, the post-docs in the Germain laboratory have done 1 representative tissue section from 2-3 mice per time point for their studies. The imaging was always designed to support findings using other techniques.
6. Access to instrumentation will also impact your n. If you have unlimited access to a scope (dream), then you can image more samples. If you are working on shared instruments, then time/access will dictate how much one can image. 
7. The instrument and configuration will also impact the number of samples and number of tissue sections per sample. I would recommend a widefield or slide scanner for high n. Confocal laser scanning microscopy is beautiful but slow. It's exciting to see advances in the field that allow more automation and batch sample preparation like the Cell DIVE and Bio Assembly Bot. 
8. Sample heterogeneity. More heterogeneous samples will need more serial sections and larger areas imaged. 
9. How rare is the event of interest? For example, if you are looking at a rare cell or phenomenon, then you might need to first calculate the frequency of the event per tissue area imaged, decide on the total number of events that need to be imaged for stats, and then calculate the expected tissue area to image. [Dr. Harikesh Wong](https://www.sciencedirect.com/science/article/pii/S0092867421006589?via%3Dihub) used 3D imaging (not highly multiplexed imaging) to count rare cells for his study. 

Best of luck with your work! 
A




![image](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/102126229/1d71c913-9508-4ff4-b797-fbddb62d96b5)

**Reply by nzangger** [reply]
Hi Andrea, 

Thank you very much for your detailed reply and all these guidelines - I will keep them in mind for the design of our studies! 

On the heterogeneity point (8) - that is definitely an important variable but can be tedious to determine. The work of [Bost et al.  ](https://www.nature.com/articles/s41592-022-01692-z) comes to mind to determine the sampling in multiplexed imaging needed for capturing the heterogeneity in various tissues. 

Looking forward to such helpful discussions in this forum! 

Thanks,
Nathan
